cmd/genji/doc/functions.go

```diff
@@ -10,12 +10,19 @@ var packageDocs = map[string]functionDocs{
 var builtinDocs = functionDocs{
 	"pk":    "The pk() function returns the primary key for the current document",
 	"count": "Returns a count of the number of times that arg1 is not NULL in a group. The count(*) function (with no arguments) returns the total number of rows in the group.",
-	"min":   "Returns the minimum value in a group.",
-	"max":   "Returns the maximum value in a group.",
-	"sum":   "The sum function returns the sum of all values in a group.",
-	"avg":   "The avg function returns the average of all values in a group.",
+	"min":   "Returns the minimum value of the arg1 expression in a group.",
```

Comment:
@asdine WDYT about this?

Reply:
I'm not a big fan of using `arg1` in the docs, but rather give parameters names that describe their purpose.
I suggest to leave the doc aspect for another PR to avoid mixing concerns during the review if that's ok 🙏🏼

Reply:
Ok, we'll need to rework the doc code a bit then, let's keep that for another one.